pkg/agent/as3/as3Utils.go

```diff
@@ -312,6 +320,13 @@ func getClass(obj interface{}) string {
 	return cl.(string)
 }
 
+func assertToBe(kind string, obj interface{}) bool {
+	if obj == nil {
+		return false
+	}
+	return (reflect.TypeOf(obj).Kind().String() == kind)
```

Comment:
reflect package adds additional overhead. Can you revert these changes from this PR.

Reply:
The impact is 'ns'(nano seconds) level, but the deployment of declaration is 's' (seconds, in fact tens of seconds) level. They are totally not in the same page. 
A bit more, assertTobe(map)  can justify not only `map[string]interface{}` but also `map[string]string` or map[string]xxxx. It may have more usage abilities.

Reply:
Benchmarking results - 

BenchmarkInterfaceTest-12 1000000000 **0.2527** ns/op 0 B/op 0 allocs/op

BenchmarkAssertToBe-12 482082412 **2.494** ns/op 0 B/op 0 allocs/op

-  reflect adds an additional overhead by a factor of 10X